src/dependencies.scala

```diff
@@ -150,5 +173,52 @@ object Dependencies {
         deps map { x => x: ClasspathDep[ProjectReference] }:_*)
     }
   }
+
+  implicit class ModuleIDOps(id: ModuleID)
+  {
+    def revMatches(other: String) = {
+      def partMatches(p: (String, String)) = p._1 == p._2 || p._1 == "+"
+      val parts = id.revision.toString.split('.')
+      val otherParts = other.toString.split('.')
+      val partsMatch = parts zip(otherParts) forall(partMatches)
+      partsMatch && (
+        parts.length == otherParts.length ||
+        (parts.length < otherParts.length) && parts.lastOption.exists(_ == "+")
+      )
+    }
+
+    def matches(other: ModuleID) = {
+      id.organization == other.organization && id.name == other.name &&
+        revMatches(other.revision)
+    }
+
+    def isAar = id.explicitArtifacts.exists(_.`type` == "aar")
```

Comment:
also discussed on irc: this won't work when aars are implicitly resolved (answer: just check all library dependencies)